src/Microsoft.ML.Recommend/MatrixFactorizationPredictor.cs

```diff
@@ -0,0 +1,403 @@
+//------------------------------------------------------------------------------
+// <copyright company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//------------------------------------------------------------------------------
```

Comment:
Licensing should change here too - check all files.

Reply:
Done.

---
In reply to: [225357793](https://github.com/dotnet/machinelearning/pull/1263#discussion_r225357793) [](ancestors = 225357793)